### PR TITLE
Create orangepi_gpio.markdown

### PR DESCRIPTION
**Pull request by pascallj**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22541

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/orangepi_gpio.markdown [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+description: "Instructions on how to integrate the GPIO capability of a Orange Pi into Home Assistant."
+date: 2019-03-29 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - DIY
+  - Binary Sensor
+  - Cover
```
Comment (klaasnicolaas) [inline source/_components/orangepi_gpio.markdown]: If you remove the cover platform, then also here 😉

Comment (pascallj) [reply source/_components/orangepi_gpio.markdown]: That's sloppy, thank you.

**Review by rohankapoorcom — APPROVED** [review]

File: source/_components/orangepi_gpio.markdown [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+description: "Instructions on how to integrate the GPIO capability of a Orange Pi into Home Assistant."
+date: 2019-03-29 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - DIY
+  - Binary Sensor
+ha_release: 0.92
```
Comment (klaasnicolaas) [inline source/_components/orangepi_gpio.markdown:13]: ⚠️ We need to change this to 0.93.
This component is not included in yesterday's cut-off.

Comment (rohankapoorcom) [reply source/_components/orangepi_gpio.markdown:13]: https://github.com/home-assistant/home-assistant.io/pull/9244

**Pull request merged**